integration_tests/on_demand_service_broker/startup_test.go

```diff
@@ -298,6 +298,40 @@ var _ = Describe("Startup", func() {
 				Expect(runningBroker.ExitCode()).ToNot(Equal(0))
 			})
 		})
+
+		Context("when disable_cf_startup_checks is set to true", func() {
+			BeforeEach(func() {
+				conf.Broker.DisableCFStartupChecks = true
+			})
+
+			It("does not fail", func() {
+				boshDirector.VerifyAndMock(
+					mockbosh.Info().RespondsWithSufficientStemcellVersionForODB(boshDirector.UAAURL),
+					mockbosh.Info().RespondsWithSufficientStemcellVersionForODB(boshDirector.UAAURL),
+				)
+
+				runningBroker = startBroker(conf)
+
+				odbLogPattern := `\[on-demand-service-broker\] \d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}\.\d{6}`
+
+				Eventually(runningBroker.Out).Should(gbytes.Say(fmt.Sprintf(`%s Starting broker`, odbLogPattern)))
+				Eventually(runningBroker.Out).Should(gbytes.Say(`-------./ssssssssssssssssssss:.-------`))
+				Eventually(runningBroker.Out).Should(gbytes.Say(fmt.Sprintf(`%s Listening on :%d`, odbLogPattern, conf.Broker.Port)))
+				Eventually(runningBroker).ShouldNot(gexec.Exit())
+			})
+
+			It("does not contact CF", func() {
+				cfAPI.Close()
+				cfUAA.Close()
+				boshDirector.VerifyAndMock(
+					mockbosh.Info().RespondsWithSufficientStemcellVersionForODB(boshDirector.UAAURL),
+					mockbosh.Info().RespondsWithSufficientStemcellVersionForODB(boshDirector.UAAURL),
+				)
+
+				runningBroker = startBroker(conf)
+				Eventually(runningBroker).ShouldNot(gexec.Exit())
+			})
+		})
```

Comment:
Please correct me if I am missing something but I think we could merge these two tests as it looks like the first test implies the second. CF API and CF UAA mocks fail if they receive unmocked calls. For instance, using only the first test and changing the `DisableCFStartupChecks` option to `false` would cause the following test failure:

```
Unmocked request:
      	GET /v2/info
      Received by:
      	mock-cloud-foundry
```

Reply:
Yes, that is true, but it is not implied specifically. We can join it in one test